components/mpas-ocean/bld/build-namelist

```diff
@@ -777,12 +776,16 @@ add_default($nl, 'config_frazil_use_surface_pressure');
 
 if ($OCN_ISMF eq 'coupled') {
 	add_default($nl, 'config_land_ice_flux_mode', 'val'=>"coupled");
+	add_default($nl, 'config_frazil_under_land_ice', 'val'=>".true.");
 } elsif ($OCN_ISMF eq 'internal') {
 	add_default($nl, 'config_land_ice_flux_mode', 'val'=>"standalone");
+	add_default($nl, 'config_frazil_under_land_ice', 'val'=>".true.");
 } elsif ($OCN_ISMF eq 'data') {
 	add_default($nl, 'config_land_ice_flux_mode', 'val'=>"data");
+	add_default($nl, 'config_frazil_under_land_ice');
```

Comment:
Just commenting that this is, indeed, what we want because the data melt fluxes include frazil.

Reply:
Any reason not to just set it to false then here?

Reply:
Were you thinking to set it to false here as a fail-safe in case we change the default value later? That sounds like a good idea so I'll do that.